app/views/shared/export/_plan_coversheet.erb

```diff
@@ -30,7 +30,9 @@
 
   <p><%= _("Last modified: ") + l(@plan.updated_at.to_date, formats: :short) %></p> <br>
 
-  <p><%= _("Copyright information:") %></p>
-  <div style="margin-left: 15px;"><p><%= _(" The above plan creator(s) have agreed that others may use as much of the text of this plan as they would like in their own plans, and customise it as necessary. You do not need to credit the creator(s) as the source of the language used, but using any of the plan's text does not imply that the creator(s) endorse, or have any relationship to, your project or proposal") %></p></div>
+  <% if @public_plan %>
```

Comment:
why are we using an instance variable instead of accessing the visibility method?